packages/gatsby-theme-api-docs/src/components/resource/method/request-response-examples.js

```diff
@@ -21,6 +22,7 @@ const RequestResponseExamples = (props) => {
 
         if (apiType.examples) {
           responsesCodeExamples.push({
+            code,
             typeDisplayName,
             value: apiType.examples[0].value,
           });
```

Comment:
This logic still looks to me like we're not actually using the example that is directly referred to in the API.   It's just pulling the examples from the type.   

I suggest to first harden that smoke test endpoint (the one named "artificially complex") with examples and modeling that have the respective use case described in the actual examples JSON and descriptions before proceeding to make sure we're not running circles.

Reply:
Yes, you are correct, as you said [here](https://github.com/commercetools/commercetools-docs-kit/pull/1333#pullrequestreview-1027864950) The code, at the moment, is ignoring the `examples` node. This means that currently we always render the example from the type.
This PR is meant to just add the response code to the header of the code example block, it doesn't add any logic regarding the handling of explicit examples.